circuits/kimchi/src/polynomials/lookup.rs

```diff
@@ -314,33 +313,41 @@ pub fn sorted<
     let by_row = lookup_info.by_row(gates);
     let max_lookups_per_row = lookup_info.max_per_row;
 
+    for t in lookup_table().take(lookup_rows) {
+        // Don't multiply-count duplicate values in the table, or they'll be duplicated for each
+        // duplicate! E.g. 3 values the same would insert the value 3 times for each instance.
```

Comment:
"3 values the same" -> typo?

Reply:
It's what I meant, but perhaps it could be clearer? If the same value `x` were to appear 3 times in the lookup table, we would count the 3, and then insert 3 `x`s for each time we encounter `x` in the table (which is 3 times). This comes into play with the dummy value, since we now iterate over the whole table and encounter it multiple times.

Any suggestions on how to express it more clearly?

Reply:
I'm actually not sure what we're trying to do exactly here. Might have to spend some time reverse engineering the protocol we're implementing :o but in english you meant "3 of the same values" no?

Reply:
I've reworded this, hopefully it's clearer now.